aikiblog/views.py

```diff
@@ -195,7 +198,6 @@ def add_stage(request):
         return render(request, 'add_stage.html', {'form': form})
 
 
-@render_to('add_techniques.html')
 def add_techniques(request):
```

Comment:
Powywalane, bo to było zdublowane, najpierw render_to a potem na końcu render. Niepotrzebne.